data/liveinst/liveinst

```diff
@@ -79,11 +79,11 @@ if [ -f /etc/system-release ]; then
     ANACONDA_PRODUCTVERSION=$(sed -r -e 's/^.* ([0-9\.]+).*$/\1/' /etc/system-release)
 fi
 
-RELEASE=$(rpm -q --qf '%{Release}' --whatprovides system-release)
```

Comment:
I more meant we can drop this one since we set the env variable elsewere now.

Reply:
ACK, will change it now.